templates/rtinfo/rtinfo.py

```diff
@@ -0,0 +1,52 @@
+from jumpscale import j
+from zerorobot.template.base import TemplateBase
+from zerorobot.template.state import StateCheckError
+
+NODE_CLIENT = "local"
+
+
+class Rtinfo(TemplateBase):
+    version = "0.0.1"
+    template_name = "rtinfo"
+
+    def __init__(self, name, guid=None, data=None):
+        super().__init__(name=name, guid=guid, data=data)
+        self.add_delete_callback(self.uninstall)
+
+    def validate(self):
+        # validate that rtinfo client is running when installed
+        self._monitor()
+
+    def _monitor(self):
+        try:
+            self.state.check("actions", "install", "ok")
+        except StateCheckError:
+            return
+
+        for s in self.node_sal.client.rtinfo.list():
+            if s.address == self.data["address"] and s.port == self.data["port"]:
+                return
+
+        self.node_sal.client.rtinfo.start(
+            self.data["address"], self.data["port"], self.data["disks"]
+        )
+
+    @property
+    def node_sal(self):
+        return j.clients.zos.get(NODE_CLIENT)
+
+    def install(self):
+        # reinstall if already present
+        try:
+            self.node_sal.client.rtinfo.stop(self.data["address"], self.data["port"])
+        except BaseException:
+            pass
+        self.node_sal.client.rtinfo.start(
+            self.data["address"], self.data["port"], self.data["disks"]
+        )
+
+        self.state.set("actions", "install", "ok")
+
+    def uninstall(self):
+        self.node_sal.client.rtinfo.stop(self.data["address"], self.data["port"])
```

Comment:
probably need to check if is present before trying to stop ? or does stop does nothing when the process doesn't exists ?

Reply:
It doesn't do anything
![image](https://user-images.githubusercontent.com/23582741/46024275-b666c680-c0e6-11e8-9257-847c395a05f8.png)